src/main/index.mjs

```diff
@@ -327,10 +338,14 @@ app.on('open-url', (event, url) => {
   log.info('Received open url event for url:' + url)
   templateId = getTemplateId(url)
   log.info('Loaded templateId:', templateId)
+  if (templateId !== undefined) {
+    log.info('send templateId:', templateId)
+    sendTemplateId(templateId)
+  }
 
   if (mainWindow) {
```

Comment:
Hi @marcopiraccini, I commented this part because it will reload the application, but it isn't necessary now with the event